src/content/blog/astro-development-with-ddev.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Astro development with DDEV"
+pubDate: 2023-11-30
+# modifiedDate: 2023-07-23
+summary: How to run Astro, Vite and other npm packages with DDEV
+author: Bernardo Martinez
+featureImage:
+    src: /img/blog/2023/10/astroonddev.png
+    alt: Astro image
+categories:
+  - Guides
+  - DevOps
+---
+
+Some projects might require you to add front-end tooling. Or the whole project might be based around client-side frameworks. DDEV provides an npm command out of the box, simplifying Docker development. I will be using this website as an example.
+
+DDEV.com is an [Astro](https://astro.build/) website. Astro is fantastic tool that packed with features that provide a low-entry barrier for developers looking to contribute features, issues and more. Among it's highlights are its low hosting cost, file-based routing, and extensive Markdown support.
+
+That said, for a while this website did not use DDEV. Adding DDEV makes it easier for those with the tool to contribute PRs, but also provides an example on how other projects can take advantage of it.
+
+As of this article, there are two ways to setup your local development on DDEV.com. With DDEV and without DDEV. Whichever is easier and more convenient for you.
+
+This guide will highlight what was required to add DDEV to an Astro project. It's meant to provide you with some background as well as tips along the way. That said, feel free to look at this [website](https://github.com/ddev/ddev.com/blob/main/.ddev/config.yaml) config first and reverse engineer your approach if that's easier.
+
+It's worth mentioning Astro uses Vite as its bundler so these steps would be very similar for any projects that might use Vite.
+
+The instructions below assume you have DDEV and Docker installed and globally available.
+
+## 1. Run `ddev config`
```

Comment:
Take a look at the local preview to see that this isn't working out - you can't use `## 1.` in both these places, doesn't do anything.